### ISSUE TITLE
Authentication failed [EU-WF][04.12.19]

### ISSUE THREAD

**Issue by Deadfail** [Levak/warfacebot#380]
After migrating the game today, the bots will not start.
![image](https://user-images.githubusercontent.com/30199368/70156442-86518500-16e6-11ea-8419-03f3c1dfd69b.png)

**Comment by SvenBM** [comment]
same here under linux, it says:

```
Connecting...<?xml version="1.0" encoding="UTF-8"?><Error Status="0" ErrorCode="0" MinVersion="1539" />
failed
```

recompile didn't help.

**Comment by Deadfail** [comment]
Authorization changed after migration.

**Comment by Levak** [comment]
Fix for wb.sh is on its way. I need to test if migration works as expected.
Then I need to update wb_launcher.hta (harder).

**Comment by Deadfail** [comment]
Good luck with the update.

**Comment by tetteykn** [comment]
> Fix for wb.sh is on its way. I need to test if migration works as expected.
> Then I need to update wb_launcher.hta (harder).

Lets us know when it fixed

**Comment by BOROH** [comment]
> Исправление для wb.sh уже в пути. Мне нужно проверить, если миграция работает, как ожидалось.
> Затем мне нужно обновить wb_launcher.hta (сложнее).

Eagerly await ))

**Comment by Levak** [comment]
Part of the fix (only shell launcher) is available at 139b03

**Comment by SvenBM** [comment]
works fine - thank you! :)

**Comment by BOROH** [comment]
> работает отлично - спасибо! :)

Works for you?

**Comment by tetteykn** [comment]
I used shell launcher fix but it didnt fix it

**Comment by Levak** [comment]
> I used shell launcher fix but it didnt fix it

And what's the issue?

**Comment by tetteykn** [comment]
Its same issue....Authentication failed

**Comment by Levak** [comment]
> Its same issue....Authentication failed

The issue in the OP is the HTA launcher, not the shell launcher.

Please provide a screenshot of your command prompt showing the issue.

**Comment by Levak** [comment]
HTA launcher fixed in 6f49e50f9a7398b2e15bb23722bdcf4f92b9a1ea
New release: https://github.com/Levak/warfacebot/releases/tag/1.0.16
